rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/LocalDataTransferServer.java

```diff
@@ -86,7 +88,12 @@ public static void startServer(Configuration conf, MetricRegistry metricRegistry
   // In embedded mode, this is called directly with local bookKeeper object
   public static void startServer(Configuration conf, MetricRegistry metricRegistry, BookKeeper bookKeeper)
   {
-    conf = new Configuration(conf);
+    try {
+      conf = new Configuration(applyRubixSiteConfig(conf));
+    } catch (Exception e) {
+      log.warn("Error in Applying Rubix site config: ", e);
```

Comment:
Hope we can get rid of this exception handling after moving to the suggested style of applyRubixSiteConfig